Lib/_pydecimal.py

```diff
@@ -2131,10 +2181,11 @@ def _power_exact(self, other, p):
             else:
                 return None
 
-            if xc >= 10**p:
```

Comment:
I have read the code above, it is not so complex, and I can guarantee that `xc` is never the power of 10.

From the start, it is made non-divisible by 10. The case `xc == 1` is handled separately. Then the only transformation of `xc` is taking the `n`th root (exact) and elevation to the `m`th degree.

Reply:
I agree - and already said the same (with different words) in comments on the PR :smile:.

In the other place the test is made, it's even easier to show that power-of-10 can't happen.

Thanks for looking at it!